react-accordion-component/src/accordian.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+export default class Accordian extends React.Component {
+  constructor(props){
+    console.log(props.dataArray);
+    super(props);
+    this.state = {display: null};
+    this.listItems = this.listItems.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.displayList = null;
+  };
+  listItems() {
+      return this.props.dataArray.map(item => {
+        const itemClass = this.state.display === item.id ? 'border margin-small ' : 'display-none';
+        return (
+          <li className="list-none" onClick={() => this.handleChange(item.id)} key={item.id}>
+            <h3 className='list'>{item.name}</h3>
+            <p className={itemClass}>{item.info}</p>
+          </li>
+        )
+      });
+  };
+  handleChange(event) {
```

Comment:
Since this takes an `id` as a parameter you should update the parameter name to reflect that, you always want variable and parameter names to be descriptive of the data they will contain